exporter.go

```diff
@@ -1650,3 +1656,113 @@ func (e *Exporter) scrapeRedisHost(ch chan<- prometheus.Metric) error {
 
 	return nil
 }
+
+func (e *Exporter) extractSentinelMetrics(ch chan<- prometheus.Metric, c redis.Conn) {
+	masterDetails, err := redis.Values(doRedisCmd(c, "SENTINEL", "MASTERS"))
+	if err != nil {
+		log.Debugf("Error getting sentinel master details %s:", err)
+		return
+	}
+
+	log.Debugf("sentinel master details : %#v", masterDetails)
+
+	for _, masterDetail := range masterDetails {
+		masterDetailMap, err := redis.StringMap(masterDetail, nil)
+		if err != nil {
+			log.Debugf("Error generating map from %s: %s", masterDetail, err)
+			continue
+		}
+
+		masterName, ok := masterDetailMap["name"]
+		if !ok {
+			continue
+		}
+
+		masterIp, ok := masterDetailMap["ip"]
+		if !ok {
+			continue
+		}
+
+		masterPort, ok := masterDetailMap["port"]
+		if !ok {
+			continue
+		}
+		masterAddr := masterIp + ":" + masterPort
+
+		sentinelDetails, err := redis.Values(doRedisCmd(c, "SENTINEL", "SENTINELS", masterName))
+		if err != nil {
+			log.Debugf("Sentinel details not found for master %s, err: %s", masterName, err)
+			continue
+		} else {
+			log.Debugf("Sentinel details for master %s: %s", masterName, sentinelDetails)
+			e.processSentinelSentinels(ch, sentinelDetails, masterName, masterAddr)
+		}
+
+		slaveDetails, err := redis.Values(doRedisCmd(c, "SENTINEL", "SLAVES", masterName))
+		if err != nil {
+			log.Debugf("Slave details not found for master %s, err:%s", masterName, err)
```

Comment:
```suggestion
			log.Debugf("Slave details not found for master %s, err: %s", masterName, err)
```

Reply:
This was updated along with other review suggestion. Thanks. :)